playground/backend/pkg/constants.go

```diff
@@ -0,0 +1,27 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkg
+
+// All application constants
+const (
+	ParentBaseFileFolder = "pkg"
+
+	JavaBaseFileFolder     = ParentBaseFileFolder + "/executable_files"
```

Comment:
I suppose it is better to move java constants to java fs file otherwise
 later there will be too many constants here (python, go, java all mixed)

Reply:
I want to keep all constant's variables in one place because these variables will be used not from the `java_fs.go` file only. If I move these constants to `java_fs.go` file like
```
...
const (
    ...
    javaExecutableFileType = "java"
    ...
)
```
I will not have available to use these constants in other places (`executors` for example) because they would be private variables, and I am not sure that it would be a good way if we will have some constants in `java_fs.go` file which would be usable in other places if we place them like public variables
```
...
const (
    ...
    JavaExecutableFileType = "java"
    ...
)
```

Reply:
Changed.